formatTest/typeCheckedTests/expected_output/patternMatching.re

```diff
@@ -116,13 +116,13 @@ let res =
     id(
       fun
       | Some(x) => x + 1
-      | None => 0
+      | None => 0,
```

Comment:
Is this trailing comma expected?

Reply:
This is a trailing comma on the function argument. I’m pretty sure this is what we want. The alternative is to not print trailing commas for functions accepting only one argument.

Reply:
I think this would be a lot nicer to look at with different `function` syntax

Reply:
Yeah, if `fun`s had braces, it would be nicer. There was discussion about adding them. In the mean time, the comma is kind of nice in that it tells you pretty clearly where to add another argument. We could also special case it so that single argument functions whose argument is a `fun |` do not print the trailing comma.

Reply:
> In the mean time, the comma is kind of nice in that it tells you pretty clearly where to add another argument

This seems like a compelling argument for keeping the trailing comma.

Reply:
It felt a bit odd when I saw it, but it makes sense.

I really like this change and I agree it helps with paren-fatigue 👍